build.fsx

```diff
@@ -828,7 +892,11 @@ Target.create "CheckReleaseSecrets" (fun _ ->
 )
 let executeFPM args =
     printfn "%s %s" "fpm" args
+    #if BOOTSTRAP
+    Shell.exec("fpm", args=args, dir="bin")
```

Comment:
This still seems to be missing (note that this is actually part of `Fake.Core.Process`)

Reply:
Ah yes, I just replace all `Shell.PascalCase` to `Shell.camelCase` this one was too much :)

Let's see if the CI is ok now